polynote-server/src/main/scala/polynote/server/NotebookSession.scala

```diff
@@ -165,7 +165,7 @@ class NotebookSession(
     subscriber.checkPermission(Permission.ReadNotebook).flatMap { _ =>
       // make sure to send status first, so client knows whether this notebook is up or not.
       sendStatus *> sendVersion *> sendNotebookContents *> sendTasks *> sendPresence *> sendCellStatuses
-    }.catchAll(err => PublishMessage(Error(0, err)))
+    }.catchAll(err => broadcastAll.publish(Error(0, err)))
```

Comment:
Hmm, this will publish the error to _everyone_ though, won't it? It should just go to the person who was denied permission.